doc/changes/changes_0.3.0.md

```diff
@@ -0,0 +1,26 @@
+# Transformers Extension 0.2.0, released 2022-10-28
+
+Code name: TBD
```

Comment:
TBD? so no release will be done right now I assume?

Reply:
new release will be done today together with another refactoring